tests/py/dynamo/conversion/test_tile_aten.py

```diff
@@ -71,5 +72,94 @@ def forward(self, x):
         )
 
 
+class TestTileConverterDynamicShape(DispatchTestCase):
+    @parameterized.expand(
+        [
+            ((3,), (3,), (6,), (1,)),
+            ((3,), (3,), (6,), (0,)),
+            ((3,), (3,), (6,), (2,)),
+            ((2,), (3,), (6,), (2, 2)),
+            ((2,), (3,), (6,), (0, 2)),
+            # 2d cases
+            ((3, 1), (3, 1), (6, 1), (0,)),
```

Comment:
is tiling with dims = (0,) same as tiling with (1,) ?

Reply:
Tiling with dims = (0,) results in that particular dimension as 0 in the tiled tensor. In the above case though it prepends -1 resulting in dims being (-1, 0) post which the dynamic input tensor is tiled